src/status_im/translations/en.cljs

```diff
@@ -604,6 +604,12 @@
    :view-cryptokitties                   "View in CryptoKitties"
    :cryptokitty-name                     "CryptoKitty #"
 
+   :view-etheremon                       "View in Etheremon"
+   :etheremon-name                       "Etheremon #"
+
+   :view-cryptostrikers                  "View in CryptoStrikers"
+   :crytpostriker-name                   "CryptoStriker #"
```

Comment:
Maybe better to use template style labels here than to concatenate strings. Same goes for `:cryptokitty-name` didn't catch it then

Reply:
Good point